dask/dataframe/io/tests/test_sql.py

```diff
@@ -11,7 +11,7 @@
 pytest.importorskip('sqlite3')
 
 
-data = """
+data = u"""
```

Comment:
The reason I didn't is because the table and column names below should be in the most typical form that users provide. It doesn't seem to make any difference, though.